app/assets/stylesheets/darkswarm/footer.scss

```diff
@@ -5,7 +5,7 @@
 footer {
   .row {
     p a {
-      font-size: 0.875rem;
+      font-size: 1rem;
```

Comment:
I wonder if we really need to set the font-size of links within paragraphs. They should inherit the same size as the paragraph.

Reply:
Thanks @sauloperez! I was wondering too - but it was there before and it works. I am happy to remove it. But then we need an #id for the "Secure and trusted" headline to still have it bigger (1.5 rem) than the text (1 rem).
Which solution do you prefer?